Kickstarter-iOS/Features/RewardsCollection/Controller/RewardsCollectionViewController.swift

```diff
@@ -285,12 +285,14 @@ final class RewardsCollectionViewController: UICollectionViewController {
     self.navigationController?.pushViewController(vc, animated: true)
   }
 
-  private func goToPledge(data: PledgeViewData) {
-    let pledgeViewController = PledgeViewController.instantiate()
-    pledgeViewController.delegate = self.pledgeViewDelegate
-    pledgeViewController.configure(with: data)
+  private func goToPledge(data _: PledgeViewData) {
+    // TODO: This class will be removed as part of our legacy checkout cleanup
 
-    self.navigationController?.pushViewController(pledgeViewController, animated: true)
+//    let pledgeViewController = PledgeViewController.instantiate()
+//    pledgeViewController.delegate = self.pledgeViewDelegate
+//    pledgeViewController.configure(with: data)
+//
+//    self.navigationController?.pushViewController(pledgeViewController, animated: true)
```

Comment:
Same here. `RewardsCollectionViewController`, is a legacy class and will be deleted. Simply commenting this code out for now.